dev-utils/karma.js

```diff
@@ -127,6 +127,12 @@ function prepareConfig(defaultConfig) {
       ') Elastic Stack ' +
       process.env.STACK_VERSION
 
+    defaultConfig.plugins.push('karma-junit-reporter')
```

Comment:
guess this line is not required. isn't it a reporter and not a plugin? https://github.com/karma-runner/karma-junit-reporter#configuration

Reply:
As far as I see it's required when running the tests locally, as I've just tried without the line `defaultConfig.plugins.push('karma-junit-reporter')` and even moving the `defaultConfig.reporters.push('junit')` line at the same position. See the below error:

```
node-puppeteer         | 15 07 2019 12:08:00.521:ERROR [reporter]: Can not load reporter "junit", it is not registered!
node-puppeteer         |   Perhaps you are missing some plugin?
```


I guess, we are now in a good position to merge it.

Reply:
you are right 👍